dodola/cli.py

```diff
@@ -356,6 +362,7 @@ def apply_qplad(
     out_zarr_region=None,
     root_attrs_json_file=None,
     new_attrs=None,
+    wet_day_post_correction=False,
```

Comment:
So it works without the hyphen in "wetday".

```suggestion
    wetday_post_correction=False,
```